localstack/services/cloudformation/models/s3.py

```diff
@@ -66,6 +66,24 @@ def convert_acl_cf_to_s3(acl):
             """Convert a CloudFormation ACL string (e.g., 'PublicRead') to an S3 ACL string (e.g., 'public-read')"""
             return re.sub("(?<!^)(?=[A-Z])", "-", acl).lower()
 
+        def transform_cfn_cors(cors_config):
+            if not cors_config:
+                return {}
+
+            transformed_cors = cors_config.copy()
+            transformed_cors["CORSRules"] = [
+                {
+                    "ID": rule["Id"] if rule["Id"] is not None else None,
+                    "AllowedHeaders": rule.get("AllowedHeaders"),
+                    "AllowedMethods": rule.get("AllowedMethods"),
+                    "AllowedOrigins": rule.get("AllowedOrigins"),
+                    "ExposeHeaders": rule["ExposedHeaders"] if rule.get("ExposedHeaders") else None,
```

Comment:
dictionary.get("key") will return None if the there's no value
```suggestion
                    "ExposeHeaders": rule.get("ExposedHeaders"),
```